app/templates/play/campaign-view.jade

```diff
@@ -138,7 +138,7 @@ if view.showAds()
                 button.btn.btn-warning.btn.btn-lg.btn-illustrated.view-solutions(data-level-slug=level.slug)
                   span(data-i18n="leaderboard.scores")
               
-              if me.get('courseInstances') && me.get('courseInstances').length
+              if me.get('role') == 'student' && me.get('courseInstances') && me.get('courseInstances').length
```

Comment:
Should use [User.isStudent()](https://github.com/codecombat/codecombat/blob/master/app/models/User.coffee#L78-78).